CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# [Unreleased]
+
+## Added
+- Added `NoBootArgsInserted` error type used when calling `as_cstring` on a `Cmdline`
+  containing only init args.
+- Added `InvalidCapacity` error type used when trying to create a `Cmdline`
+  either with zero capacity (using `new` or `try_from`) or with smaller capacity
+  than required for arguments (using `try_from`).
+- Added `insert_init_args` method allowing insertion of init arguments into `Cmdline`.
```

Comment:
We should also say we added `try_from` that has support for both init args and command line args.